### ISSUE TITLE
Linting CSS/JS

### ISSUE THREAD

**Issue by maudetes** [etalab/data.gouv.fr#392]
Créé par @abulte

**Comment by agarrone** [comment]
@maudetes Tjr d'actu ?

**Comment by maudetes** [comment]
Oui, je pense qu'il faudrait qu'on repriorise d'ailleurs, cc @nicolaskempf57

**Comment by bolinocroustibat** [comment]
We could use https://biomejs.dev/ which replaces eslint and prettier and which is pretty fast, but not sure it works with node 6!

**Comment by agarrone** [comment]
Fait
